orttraining/orttraining/training_api/optimizer.cc

```diff
@@ -17,23 +17,14 @@ namespace api {
 
 namespace {
 
-// Currently all parameters are in a single group, so we hardcode group0 here.
 constexpr char GROUP_ZERO_NAME[] = "group0";
+static std::string FullQualifiedName_AdamWOptimizer(std::string(kMSDomain) + ":AdamWOptimizer");
+static std::string FullQualifiedName_SGDOptimizerV2(std::string(kMSDomain) + ":SGDOptimizerV2");
```

Comment:
optional: can be constexpr char[].

Reply:
The tricky thing is we are doing concat here during initialization, tried few approaches don't work. The otherhand, I find the related logic can avoid such a global var, so those two vars are removed.